app/scripts/directives/score.directive.js

```diff
@@ -27,13 +27,13 @@ angular.module('scoreTrackerApp')
 
         scope.removeScore = function(uuid) {
           ScoreService.remove(uuid);
-          element.parent().remove();
-          scope.updateSummary();
+          element.remove();
```

Comment:
True.  The `ng-repeat` will handle the remove once the `on-change` callback refreshes the scores (by calling `getList`).  However, that relies on the directive being called with an `on-change` callback that updates the list of scores.  I can picture a use case where the directive might be used without such a callback and in that case I think the directive should do this work.

In the end, this directive has more work before it would be truly re-usable.